.gitignore

```diff
@@ -16,6 +16,7 @@
 # under the License.
 
 build/
+install/
```

Comment:
Why this? Is this generated by some IDE?

Reply:
No, since we specify `CMAKE_INSTALL_PREFIX` in cmake, I think someone might specify `CMAKE_INSTALL_PREFIX=install` in the repo directory as well.

Reply:
> No, since we specify `CMAKE_INSTALL_PREFIX` in cmake, I think someone might specify `CMAKE_INSTALL_PREFIX=install` in the repo directory as well.

I doubt anyone would do this, if you really need to install in src tree, you can specify `CMAKE_INSTALL_PREFIX=/build/install`.